.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce behavior
```

Comment:
Also, we could add one for including screenshots: `### Screenshots (if relevant)`